labs/splice.md

```diff
@@ -3,81 +3,111 @@ layout: default
 title: Splice
 parent: Labs
 ---
-# Splice
+# Hyperledger Splice
```

Comment:
Labs cannot be named liked this. Please see [Labs Representation](https://labs.hyperledger.org/proposal.html#labs-representation) for more information. Please update this pull request to remove the "renaming" of the lab.

Reply:
Done

Reply:
Hi @tkuhrt, sorry for that, my mistake. fixed.